src/core/Akka.Remote/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Google.ProtocolBuffers" version="2.4.1.555" targetFramework="net45" />
-  <package id="Helios" version="1.4.1" targetFramework="net45" />
+  <package id="Helios" version="2.0.1" targetFramework="net45" />
```

Comment:
That's correct - bringing upgrades to Akka.Remote itself soon.

All of the old Helios APIs are still present in 2.0 and that's what the rest of the system is still currently using.